### DISCUSSION TITLE
How to disable search

### DISCUSSION THREAD

**Discussion by AtlantisDe** [mkdocs/mkdocs#2724] [Q&A]
### Description

---
Hi, I want to disable "search", how to do ?

**Comment by ribafs** [comment]
This

.nav-item{
    display: none;
}

Hide search and github link

Create
extra.css

extra_css:
  - extra.css

**Comment by pawamoy** [comment]
Edit: fixed my answer thanks to @fu-sen (empty key gives a dict, not a list, so added empty list `[]`).

I believe you can simply define an empty `plugins: []` key in `mkdocs.yml`.
If you need to configure plugins though, just omit the `search` plugin from the list:

```yaml
plugins:
# no search plugin
- someplugin
```

**Reply by fu-sen** [reply]
Well, until recently I had search problems when using MkDocs in Japanese, so I used this method. https://github.com/mkdocs/mkdocs/pull/2178

**Comment by fu-sen** [comment]
if there is no plugin:

```
plugins: []
```
